events/seizing_collasuyu.txt

```diff
@@ -0,0 +1,63 @@
+﻿namespace = seizing_collasuyu
+
+# Seizing the southern ranges JE completion
+seizing_collasuyu.1 = {
```

Comment:
Can you make new custom events start with dvg_ in the file name? Makes it easier when we need to apply vanilla patches on the mod.

Reply:
Oh I didn't know you could resolve it here so manually changed things, I think its fixed now?